### ISSUE TITLE
App webpack issue? Unable to use sample from codesandbox (no animation/bad reorder)

### ISSUE THREAD

**Issue by rosskevin** [atlassian/react-beautiful-dnd#332]
## Bug or feature request?
Bug - not sure where but I suspect it is a difference in my build that I cannot identify

### Expected behavior
I can use the sample code within my webpacked build: https://codesandbox.io/s/4jw99m4n20

### Actual behavior
No animation, and incorrect reordering - notice I drag 9 but 0 moves down.

### Steps to reproduce
Tough one - I used exactly the sample code, with a route directly to it.  I am not wrapping it in any components whatsoever.  I stripped everything else from the `head` as pictured.  It is as stripped down as can be - so I suspect a build issue.  

![dnd2](https://user-images.githubusercontent.com/136564/36285728-669c9a08-1272-11e8-9397-baaeef2edd66.gif)

### Browser version
`66.0.3346.0 (Official Build) canary (64-bit)`

### Demo
Original sandbox code with updated dependencies: https://codesandbox.io/s/4jw99m4n20 works fine, but not with my build.

## TL;DR
Any thoughts/directions I should explore?  I'm stuck since I took out absolutely everything from our app (with the exception of routing) but it renders this sample code directly. 

I'm a material-ui committer and I'd like to get a sortable  `List` example created for 1.0.0 beta docs.

**Comment by alexreardon** [comment]
Without seeing your project or config it is hard to know..

**Comment by alexreardon** [comment]
It sounds like a ref is not being bound. Any console warnings?

**Comment by rosskevin** [comment]
No console warnings, we are on react 16.2, but the codesandbox is as well. 

I'll try to harvest a resulting webpack. We have no css loaders (use jss), use `ts-loader` for typescript and `babel-loader` with `preset-env` for polyfills. I have to think it a polyfills problem.

If any of this sparks an idea, please let me know, otherwise I'll grab webpack and try some debugging on ref.

**Comment by rosskevin** [comment]
Here's a webpack config and babel setup:
https://gist.github.com/rosskevin/019c8020aa4775b5730f6dac47b96d8d

If you see anything please let me know.  I'll try to debug the `ref`.

**Comment by rosskevin** [comment]
I'm not sure what refs I am looking for but peeking through the react dev tree looks good from my uninformed perspective.

I've ruled out everything else in my App code - the index now renders only the sample code:
![raw](https://user-images.githubusercontent.com/136564/36314545-c4dd1060-12fa-11e8-9a87-f8e7c31fb285.png)

Same issue presents, so I assume it is caused by:
- webpack config difference
- dependency difference

I'll pare down the dependency tree next.

**Comment by rosskevin** [comment]
Confirmed not a dependency issue from my app - definitely down to webpack/typescript/babel 7 preset-env polyfills.  My package.json is minimal now with:

```
  "dependencies": {
    "react": "16.2.0",
    "react-beautiful-dnd": "^4.0.1",
    "react-dom": "16.2.0"
  },
```

Will keep narrowing this down.  I'm starting to think something babel 7/polyfill related?

**Comment by rosskevin** [comment]
I've got a minimal reproduction (typescript):
https://github.com/rosskevin/issue-react-beautiful-dnd-example

Along the way - I noticed that I had not copied my `.babelrc` to this sample project, yet it was serving.  So my assumption at this point is that I have messed up polyfills since I expected to have webpack use `ts-loader -> babel` for polyfilling.

Looking into that now.

**Comment by rosskevin** [comment]
Ok @alexreardon - I think I've boiled this down so as to not waste anyone else's time.

This repo has `@babel/preset-env` setup and `babel-loader` is now run after `ts-loader`.  Strangely enough, I see `Based on your code and targets, none were added.` so perhaps I don't have a polyfill issue after all, but I still see the same exact behavior.

This is a minimal typescript + babel project with the sample code (`master`):
https://github.com/rosskevin/issue-react-beautiful-dnd-example

I'll next try to convert it to non-typescript (on a branch) to see if I get the same result.

**Comment by rosskevin** [comment]
Here is the `vanilla-javascript` branch - still the same thing. I think this is the simplest possible reproduction with copy/paste from the working codesandbox example:

https://github.com/rosskevin/issue-react-beautiful-dnd-example/tree/vanilla-javascript

**Comment by rosskevin** [comment]
Mystery solved: 
https://github.com/rosskevin/issue-react-beautiful-dnd-example/commit/b76e67f6f7bac694bc55b22422ad9c26a0a1e65a

Total user error.  I had started with the README link to the sandbox, then at some point had searched codesandbox.io for other examples.  Thinking I was going back to the original from the README - I was _actually_ on a bad or old fork of the original example.

Wow.  I colossally wasted some time on this one!

**Comment by alexreardon** [comment]
Thank you for following this through to the end. I am glad the solution was an easy one!!
